src/ucp/api/ucp.h

```diff
@@ -1223,6 +1224,12 @@ typedef struct {
      * Pointer to user data passed to callback function.
      */
     void          *user_data;
+
+    /**
+     * Reply buffer. Can be used for storing operation result, for example by
```

Comment:
Can you please expand what is "storing" operations ?  Specifically what I'm missing is what get stored in the buffer and how it can be used.